wire-ios/WireUITests/Pages/TeamCreationStepsPage.swift

```diff
@@ -0,0 +1,70 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+
+/// Team creation pages from user account
+class TeamCreationStepsPage: PageModel {
+
+    override var pageMainElement: XCUIElement {
+        continueButton
+    }
+
+    var teamNameTextField: XCUIElement {
+        let elementsQuery = app.textFields
+        return elementsQuery["Your Team"]
+    }
+
+    var continueButton: XCUIElement {
+        let elementsQuery = app.scrollViews.otherElements
+        return elementsQuery.buttons["Continue"]
```

Comment:
Question: Can this follow same format as the the checkbox method below (as in, not filtering the type and using just the ID)?